pubsub/gcppubsub/example_test.go

```diff
@@ -0,0 +1,107 @@
+// Copyright 2018 The Go Cloud Authors
```

Comment:
Please update this to Go Cloud Development Kit Authors.

Reply:
Done.